apps/dotcom/client/src/tla/components/TlaSidebar/components/TlaSidebarFileLink.tsx

```diff
@@ -98,11 +97,7 @@ export function TlaSidebarFileLinkInner({
 
 	useEffect(() => {
 		// on mount, trigger rename action if this is a new file.
-		if (
-			(state?.mode === 'create' || state?.mode === 'duplicate') &&
-			isActive &&
-			focusCtx.shouldRenameNextNewFile
-		) {
+		if (isActive && focusCtx.shouldRenameNextNewFile) {
```

Comment:
Couldn't find any issues with omitting the mode in this check, but maybe you can double check?